runtime/interpreter/encoding_test.go

```diff
@@ -463,6 +511,71 @@ func TestEncodeDecodeComposite(t *testing.T) {
 		testEncodeDecode(t,
 			encodeDecodeTest{
 				value: expected,
+				encoded: []byte{
```

Comment:
Same as above, the diff is inferred incorrectly, the actual change is duplicating the existing test and making it decode-only (to test backwards-compatibility), and adjusting the existing test to store the qualified identifier instead of the type identifier